packages/shared/src/utils/digital-options/digital-options.ts

```diff
@@ -0,0 +1,42 @@
+import { isEuCountry } from '../location';
+
+type TMessage = {
+    title: string;
+    text: string;
+    link: string;
+};
+
+type TShowError = {
+    message: string;
+    header: string;
+    redirect_label: string;
+    redirectOnClick: null;
+    should_show_refresh: boolean;
+    redirect_to: string;
+    should_clear_error_on_click: boolean;
+};
+
+export const showDigitalOptionsUnavailableError = (showError: (t: TShowError) => void, message: TMessage) => {
+    const { title, text, link } = message;
+    showError({
+        message: text,
+        header: title,
+        redirect_label: link,
+        redirectOnClick: null,
+        should_show_refresh: false,
+        redirect_to: '/mt5',
+        should_clear_error_on_click: true,
+    });
+};
+
+type TAccounts = {
```

Comment:
@niloo-fs please put all type declarations to the top of the file as per our TS convention 🙂